### DISCUSSION TITLE
Minify code create issue with v5

### DISCUSSION THREAD

**Discussion by borisdamevin** [webpack/webpack#14576] [Q&A] [answered]
Hi,

I have a problem with javascript minify and one external plugin. For make simple:

Before:

`timestamp = countdown_default()(new Date(datetime + 'Z'), Date.now(), countdown_default.a.DAYS);`

* `webpack 4` 
* `terser-webpack-plugin 4`

After: 

`timestamp = countdown_default()(new Date(datetime + 'Z'), Date.now(), (countdown_default()).DAYS);`

* `webpack 5` 
* `terser-webpack-plugin 5`

My problem is the code `countdown_default.a.DAYS` is change to `(countdown_default()).DAYS)` and this create an error and the plugin is not launch. I just know, the issue came after the webpack 5 update, and nothing more.

Webpack et Terser have change something for generate this code? Someone has already get this issue ?

Thank you by advance !

**Comment by alexander-akait** [comment, marked as answer]
Try to update `terser`, if it will not help, please open an issue in `terser` repo with example of code

**Reply by borisdamevin** [reply]
Thank you for advice but this change nothing :( . I made a ticket on `terser`. https://github.com/terser/terser/issues/1099
